node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerOperationsImpl.java

```diff
@@ -323,7 +300,12 @@ public void stopServicesOnNode(ContainerName containerName) {
     }
 
     @Override
-    public Optional<Docker.ContainerStats> getContainerStats(ContainerName containerName) {
+    public void trySuspendNode(ContainerName containerName) {
+        executeCommandInContainer(containerName, nodeProgram, "suspend");
```

Comment:
This changes suspend failures from non-fatal to fatal, why?

Reply:
I wasn't sure what to do with this one, none of the other nodectl commands are handled this way. I added checking for when container is missing the only place this is called: https://github.com/vespa-engine/vespa/pull/6998/files#diff-22e52ac72910bf612ee17481b6d4d5c8R298
Additionally, there is a try/catch on Exception here: https://github.com/vespa-engine/vespa/pull/6998/files#diff-22e52ac72910bf612ee17481b6d4d5c8R341

Reply:
Stop services won't be called if suspend fails, with this change. That seems unwarranted. The suspension is just an optimization to make it faster to start again afterwards.